style.css

```diff
@@ -1,554 +1,1111 @@
-/* General selections */
+
 * {
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
-:root {
-  --navcolor: #3c3a39;
-  --txtcolor: #fff;
-  --txtcolor2: #172b4d;
-  --maincolor: #ff6b00;
-  --bkcolor: #1c1a19;
-  --line: #212121;
-  --colorb: #ffffff3d;
-  --backc: #fbf8f7;
-  --btncol: #f4f1f0;
-  --btncol2: #fff5e1;
+body {
+  padding: 0;
+  margin: 0;
 }
 
-body {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
+/* styling the navbar */
+.navbar-container {
+  background-color: #3c3a39;
+  color: white;
+  padding: 7px 24px;
+  height: 48px;
+}
```

Comment:
- [ ] Please 🙏🏽 make sure that the navbar completes all the width, for a better user experience.

- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

![image](https://user-images.githubusercontent.com/58642949/187038438-68f0c630-a0da-4326-8394-535ad057ec8c.png)